src/app/catalog/product/components/product-detail/product-detail.component.html

```diff
@@ -7,12 +7,23 @@
   <div class="row">
     <div class="product-details" *ngIf="product$ | async as product">
       <img class="product-img" src="{{ product.image }}" alt="product image" />
+
       <div class="product-description">
         <h2>{{ product.title }}</h2>
+
         <h3>Price</h3>
         <p>{{ product.price }}</p>
+
+        <div>
+          <h3>Number of funnels</h3>
```

Comment:
I don't think "funnel" is the right word here.
I'd rather write "Number of items".

Reply:
Fixed.